### PR TITLE
Null spark executor issue resolved

### LINKED ISSUE #903: [BUG] - Spark Executor table Empty
Multi-workspace deployments seem to have some issue pushing the case sensitivity setting into the thread-local sparkSession.

This can be seen in #819 

Additionally, the Spark Executor table is empty -- initial analysis points to the Timestamp field not being correctly cased resulting in null Timestamps resulting in no data making it to the targets.

### PR DESCRIPTION
**Pull request by sriram251-code**
closes #903

### CODE REVIEW

**Review by souravbaner-da — CHANGES_REQUESTED** [review]
@Sriram-databricks    Please remove unnecessary debug print statement

**Review by souravbaner-da — APPROVED** [review]

File: src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala [hunk 1]
```diff
@@ -778,6 +781,8 @@ trait BronzeTransforms extends SparkSessionWrapper {
           }
         }
 
+       val baseEventsDF = baseEventsDFRaw.withColumnRenamed("DerivedTimestamp","Timestamp")
+         logger.log(Level.INFO, s"SPARK CASE SENSITIVITY - POS1: THREAD: ${Thread.currentThread()} --> ${sparkCaseSensitiveValueDebug()}")
```
Comment (GeekSheikh) [inline src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala:785]: these 4 POS logs were just for debugging, let's remove these.

File: src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala [hunk 2]
```diff
@@ -761,8 +763,9 @@ trait BronzeTransforms extends SparkSessionWrapper {
           } else col("Timestamp")
 
           baseDF
-            .withColumn("Timestamp", fixDupTimestamps)
+            .withColumn("DerivedTimestamp", fixDupTimestamps)
```
Comment (GeekSheikh) [inline src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala:766]: This is a great idea. I'm thinking we might should do something similar for all the complex field derivations. The stageId derivation could be getting null for a similar reason.

What I mean by something similar is give them a temporary column name and then rename them, drop the originals and rename it back to the target name.

* executorIdOverride
* progressCol
* stageIDColumnOverride
* stageAttemptIDColumnOverride

**Review by GeekSheikh — CHANGES_REQUESTED** [review]

**Comment by sriram251-code** [comment]
duplicate of https://github.com/databrickslabs/overwatch/pull/940.
I have moved the review comments to 940 PR.

Closing it.

**Pull request closed without merging**